content/hi/docs/reference/glossary/image.md

```diff
@@ -0,0 +1,17 @@
+---
+title: इमेज(Image)
+id: image
+date: 2018-04-12
+full_link: 
+short_description: >
+  यह एक कंटेनर का एक संग्रहीत उदाहरण है और इसमें एप्लिकेशन चलाने के लिए आवश्यक सॉफ़्टवेयर का एक सेट होता है।
+
+aka: 
+tags:
+- fundamental
+---
+एक {{< glossary_tooltip text="कंटेनर" term_id="container" >}}  का संग्रहित इंस्टेंस जिसमें किसी एप्लिकेशन को चलाने के लिए आवश्यक सॉफ़्टवेयर का एक सेट होता है।
```

Comment:
both the short description and fundamental are same in the original rnglish content. 

I suggest we use the second option, without the word किसी and fixed spelling for संग्रहित = संग्रहीत

so :  एक {{< glossary_tooltip text="कंटेनर" term_id="container" >}} का संग्रहीत इंस्टेंस जिसमें एप्लिकेशन को चलाने के लिए आवश्यक सॉफ़्टवेयर का एक सेट होता है।